CONTRIBUTING.md

```diff
@@ -48,21 +48,11 @@ $ bin/phpunit
 ```
 
 This command will only run tests which does not require a real web server. To run the full tests, you will need to
-copy/paste the `phpunit.xml.dist` to `phpunit.xml` and uncomment the `TEST_SERVER` php server variable. Then, you have
-three possibilities:
-
- * Rely on Apache2 (PHP) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/apache2-php.conf`.
- * Rely on Apache2 (HHVM) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/apache2-hhvm.conf`.
- * Rely on Nginx (PHP and HHVM) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/nginx.conf`.
-
-I would recommend you to rely on Apache2 as Nginx does not support TRACE request. When, your web server is configured
-and running, you will need to increase the number of fastcgi child processes (otherwise the main process will crash)
-and then, start it with the following commands:
+copy/paste the `phpunit.xml.dist` to `phpunit.xml` and uncomment the `TEST_SERVER` php server variable. Then, just 
```

Comment:
If you'd like to, have a look at my .travis.yml in another project, where I added Scrutinizer as an environment variable and execute the coverage report on one nose only in combination with the allow_failures and fast_finish directives

https://github.com/kreait/firebase-php/blob/master/.travis.yml
(the project uses the HTTP Adapter as well, btw :) )